### ISSUE TITLE
Customize colors?

### ISSUE THREAD

**Issue by jstetten** [veggiedefender/hn-friends#6]
Hey, nice addon! I've been waiting for someone to do this for Quantum.

Do you know a way to allow users to choose their own highlighting color? Even just a text field for custom CSS would suffice for the time being.

Thanks!

**Comment by veggiedefender** [comment]
Makes sense. Right now there is no way to change the color. What do you think of just copying the color from the top bar (which is adjustable in HN settings)?

**Comment by jstetten** [comment]
That's a very sensible idea.

Since I apply my own custom CSS to Hacker News, I simply overrode the `a[href^="user?"].friends` selector you inserted for now.
